lib/src/tar/tar_file.dart

```diff
@@ -153,18 +151,18 @@ class TarFile {
 
     final headerBytes = header.getBytes();
 
-    // The checksum is calculated by taking the sum of the unsigned byte values
-    // of the header record with the eight checksum bytes taken to be ascii
-    // spaces (decimal value 32). It is stored as a six digit octal number
-    // with leading zeroes followed by a NUL and then a space.
+    /// The [checksum] is calculated by taking the sum of the unsigned byte
```

Comment:
Should dartdoc comments be internal to the function like this? What purpose does that serve?

Reply:
It is a bit more pleasant to read when browsing code on Github, not very important though

Reply:
I'd err on consistency and leave dartdoc comments for api documentation purposes...which is obviously lacking in this library :-).